### ISSUE TITLE
Support relative paths for FOLDER_ANNOTATION

### ISSUE THREAD

**Issue by diversit** [OmegaVVeapon/kopf-k8s-sidecar#29]
Running into this issue when trying to replace the default Grafana sidecar by this one.

The Kiwigrid sidecar supports the path in the FOLDER_ANNOTATION value to be relative.
Its documentation says:
```
The annotation the sidecar will look for in configmaps to override the destination folder for files. 
The annotation value can be either an absolute or a relative path. Relative paths will be relative to FOLDER.
```

This sidecar does not seem to support relative paths:
```
[2021-11-05 13:03:44,649] kopf.objects         [ERROR   ] Failed to write file kubernetes/scheduler.json
```

I prefer to use relative paths so teams adding files via ConfigMaps don't have to be bothered with the actual path where the file is being copied. E.g. `/tmp/dashboards` in case of Grafana dashboards.

**Comment by OmegaVVeapon** [comment]
Fair request, should be fairly trivial to add.

**Comment by OmegaVVeapon** [comment]
Fix present in the latest release [1.4.0](https://github.com/OmegaVVeapon/kopf-k8s-sidecar/releases/tag/1.4.0).

Thanks for the opening the issue @diversit !
